src/styles/main.scss

```diff
@@ -0,0 +1,122 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  min-height: 100vh;
+}
+.tetris {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  position: relative;
+}
+
+.tetris__header {
+  font-family: 'Inter', sans-serif;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 58.09px;
+  text-align: center;
+  width: 806px;
+  height: 83px;
+  position: absolute;
+  top: 56px;
+  left: 244px;
+}
+
+.tetris__body {
+  width: 100%;
+  height: 500px;
+  background-color: #ffffff;
+  margin-top: 20px;
+  align-items: center;
+  justify-content: center;
+}
+
+.rectangle__box {
+  width: 200px;
+  height: 400px;
+  position: absolute;
+  top: 200px;
+  left: 250px;
+  border: 1px solid #000000;
+}
```

Comment:
As I mentioned in the HTML file, this should be `.tetris__box`,
and it should not contain position: absolute and the top and left properties, you can just use the width and the height and the border property

and for the middle box you can add a modifier as I mentioned before: 
```
.tetris__box_border-width_3 {
    border-width: 3px;
  }
```